### PR TITLE
chore(sentinel-app): rename to `jsonrpc-app`

### PR DESCRIPTION
**Pull request by gitmp01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,152 +1,107 @@
...
+```bash
+curl \
+  -X POST \
+  -H 'Content-Type: application/json' \
+  -d '{"jsonrpc":"2.0","id":1,"method":"process","params":["bsc", "38745426", "false", "true"]}' \
+  http://127.0.0.1:3030/v1/rpc
+```
 
-## :black_nib: To Do:
+The output would contain all the signature for the configured events in the `signed_events` array:
```
Comment (envin3) [inline README.md]:
```suggestion
The output would contain all the signatures for the configured events in the `signed_events` array:
```

**Review by envin3 — COMMENTED** [review]
Just a small comment above.

File: README.md [hunk 2]
```diff
@@ -1,152 +1,107 @@
-# :closed_lock_with_key: pTokens Core
+# Event Attestator
 
-The Provable __pTokens__ core which manages the cross-chain conversions between native and host blockchains.
+**Note:** this repo is currently a working in progress and it will be update in the incoming days
```
Comment (ubordignon) [inline README.md]: `working` -> `work`

**Review by ubordignon — COMMENTED** [review]
Wording suggestion

**Pull request merged**